widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/base/Button.kt

```diff
@@ -0,0 +1,33 @@
+package com.glia.widgets.view.unifiedui.config.base
+
+import android.os.Parcelable
+import androidx.annotation.ColorInt
+import com.google.gson.annotations.SerializedName
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class Button(
```

Comment:
Note from me: It would be very nice if we could have a separate 'internal' package in widget SDK same as in Core SDK. What do you think? If we can agree on that then I will add a ticket to the backlog.
@DavDo @andrews-moc @AndriiHorishniiMOC

Reply:
Maybe. Haven't thought about this.

Reply:
Do you mean to move the remote config to a separate module?